src/vm/gcenv.ee.cpp

```diff
@@ -1349,3 +1349,39 @@ void GCToEEInterface::HandleFatalError(unsigned int exitCode)
 {
     EEPOLICY_HANDLE_FATAL_ERROR(exitCode);
 }
+
+bool GCToEEInterface::ShouldFinalizeObjectForUnload(AppDomain* pDomain, Object* obj)
+{
+    MethodTable *pMT = obj->GetMethodTable();
```

Comment:
Ok! Are we still using `#ifndef FEATURE_CORECLR` for this, or should I replace the body of this method with `return true`?